var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -65,6 +65,10 @@ class LlvmAmdgpu(CMakePackage):
     # This is already fixed in upstream but not in 4.2.0 rocm release
     patch('fix-spack-detection-4.2.0.patch', when='@4.2.0:')
 
+    # Add LLVM_VERSION_SUFFIX
+    # https://reviews.llvm.org/D115818
+    patch('llvm-version-suffix-macro.patch', when='@:4.3.2')
```

Comment:
apply always, whenever AMD forks from the llvm that releases with this patch add limit